src/BarChart/components/BarChartContainer.ts

```diff
@@ -56,6 +69,9 @@ export default class BarChartContainer extends Component<BarChartContainerProps,
         if (this.subscriptionHandle) {
             mx.data.unsubscribe(this.subscriptionHandle);
         }
+        if (this.intervalHandler) {
+            clearInterval(this.intervalHandler);
```

Comment:
Why? This clears the interval when the props change but does not re-create it. So for example in the listening to grid, once a user selects a different context, the chart stops refreshing.

Reply:
Added a function call on component will receive props